lambda/index.py

```diff
@@ -49,7 +49,7 @@ def _get_paginated_collection(function, key, page_token=None, collection=None):
     collection += response[key]
 
     if 'nextPageToken' in response:
-        get_paginated_collection(function, key, collection=collection,
+        __get_paginated_collection(function, key, collection=collection,
```

Comment:
I believe it's just prefixed with one underscore - can you ensure the typo fix here is correct?